zip-0320.rst

```diff
@@ -5,7 +5,242 @@
+   var t1 = bs58check.encode(Buffer.concat([Uint8Array.from([28, 184]), pkh2]))
+   console.log(t1)
+
+Specification (Alternative 2)
+=============================
+
+Traceable Unified Addresses
+---------------------------
+
+A Traceable Unified Address is a reencoding of a transparent Zcash address into
+a Unified Address [#zip-0316-unified-addresses]_. 
+
+Motivations for Alternative 2
+-----------------------------
+
+Traceable Unified Addresses fit into the existing Zcash Unified Address
+ecosystem. As such, wallets that support Unified Addresses will be able to
+parse (but not necessarily send to) a Traceable Unified Address. Even in the
+case that Traceable Receivers are not understood by the sending wallet, a
+Unified Address-supporting wallet will be able to automatically provide good
+error messages for their users to indicate that the wallet needs to be updated
+to understand these addresses.
+
+In addition, by integrating with the Unified Address framework, it becomes
+possible for the addresses being generated to include extra metadata, in
+particular, metadata items such as an Address Expiry Height or Address Expiry
+Date [#zip-0316-address-expiry]_ may be included. For exchange use cases such
+as Binance's, it is useful to ensure that an address provided to a user has a
+limited utility life, such that after expiration the user must obtain a new
+address in order to be able to continue to send funds.
+
+Detailed Specification (Alternative 2)
+--------------------------------------
+
+Upon activation of this ZIP, the section `Encoding of Unified Addresses` of ZIP
+316 [#zip-0316-unified-addresses]_ will be modified to define a new
+Traceable Receiver Type having typecode :math:`\mathtt{0x04}`, the value of
+which MUST be the 20-byte **validating key hash** of a Zcash P2PKH Address as
+defined in [#protocol-taddr]_.
+
+The "Requirements for both Unified Addresses and Unified Viewing Keys" section
+of ZIP 316 [#zip-0316-unified-requirements]_ will be modified as follows::
+
+  A Unified Address or Unified Viewing Key MUST contain at least one
+  shielded Item (Typecodes :math:`\mathtt{0x02}` and :math:`\mathtt{0x03}`).
+  The rationale is that the existing P2SH and P2PKH transparent-only
+  address formats, and the existing P2PKH extended public key format,
+  suffice for representing transparent Items and are already supported
+  by the existing ecosystem.
+
+will be replaced by::
+
+  A Unified Address MUST contain at least one Receiver, plus any number
+  of Metadata Items. The selection of Receivers is further restricted 
+  such that a Unified Address MUST **either** contain at least one shielded
+  Receiver (Typecodes :math:`\mathtt{0x02}` and :math:`\mathtt{0x03}`), OR 
+  MUST contain **only** a Traceable Address Receiver (Typecode
+  :math:`\mathtt{0x04}`).
+
+  A Unified Viewing Key MUST contain at least one shielded Item (Typecodes
+  :math:`\mathtt{0x02}` and :math:`\mathtt{0x03}`). 
```

Comment:
This means that there is no UIVK or UFVK from which deriving a Unified Address will produce a Traceable UA. That seems fine; just making sure that it is intended.